code/game/objects/items/devices/traitordevices.dm

```diff
@@ -68,31 +68,26 @@ effective or pretty fucking useless.
 		Wavelength is also slightly increased by the intensity as well.
 */
 
-/obj/item/device/rad_laser
-	name = "health analyzer"
-	icon_state = "health"
-	item_state = "analyzer"
-	desc = "A hand-held body scanner able to distinguish vital signs of the subject."
-	flags = CONDUCT
-	slot_flags = SLOT_BELT
-	throwforce = 3
-	w_class = 1
-	throw_speed = 3
-	throw_range = 7
+/obj/item/device/healthanalyzer/rad_laser
+	desc = "A hand-held body scanner able to distinguish vital signs of the subject. It has a strange device hooked up to its scanning end."
```

Comment:
I too see no reason to add it back, the hardtells for traitor items was a silly idea.

Traitors should be discovered via their own choice of actions, not because someone was ocd enough to memorise the descriptions of the items in use.